plugin-niassa/src/main/java/ca/on/oicr/gsi/shesmu/niassa/NiassaServer.java

```diff
@@ -628,7 +631,15 @@ public void configuration(SectionRenderer renderer) {
   }
 
   public WorkflowRunEssentials directoryAndIni(long workflowRun) {
-    return directoryAndIniCache.get(workflowRun).orElse(WorkflowRunEssentials.EMPTY);
+    WorkflowRunEssentials wre =
+        directoryAndIniCache.get(workflowRun).orElse(WorkflowRunEssentials.EMPTY);
+    if (wre == WorkflowRunEssentials.EMPTY) {
+      definer.log(
+          String.format(
+              "NiassaServer.DirectoryAndIniCache.fetch: Niassa Workflow Run fetch failed for WorkflowRun SWID: %d"),
```

Comment:
Is the String.format call intended to have the `workflowRun` accession as its second argument?